### ISSUE TITLE
Do not start with corrupted shortcuts

### ISSUE THREAD

**Issue by karussell** [graphhopper/graphhopper#1038]
We observed that it was possible that the import failed to complete the CH preparation (using a version before #980) and import these corrupted (?) shortcuts to successfully start. It then served the results as if there was no CH preparation (all edges had the same unprepared level)

**Comment by boldtrn** [comment]
Does this issue still happen? Is there a way to reproduce this?

**Comment by karussell** [comment]
GraphHopper should not produce corrupt shortcuts anymore but in case of a new bug it could and would then load these without reporting a problem or stopping. To reproduce the following should work: create a graph with multiple CH profiles in parallel via an old version and provide too few RAM and then some CH preparations will stop with OutOfMemory errors or similar and then load these files with a recent version.

**Comment by karussell** [comment]
Or probably just "removing" the shortcuts via `echo '' > shortcuts_fastest_car` could work to reproduce it (or create an empty shortcuts files via GH code somehow)

**Comment by karussell** [comment]
The shortcuts were just empty and not corrupt. We could throw an exception when loading empty shortcuts but then shortcuts must be created and cannot be postponed. Not sure if this is an issue though. I'll close for now.
